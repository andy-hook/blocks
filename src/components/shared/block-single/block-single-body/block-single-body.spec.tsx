import React from "react"
import BlockSingleBody from "./block-single-body"
import { renderWithTheme } from "@test-utils"

describe("<BlockSingleBody />", () => {
  it("renders correctly", () => {
    const tree = renderWithTheme("dark", <BlockSingleBody />)
    expect(tree).toMatchSnapshot()
  })
})
