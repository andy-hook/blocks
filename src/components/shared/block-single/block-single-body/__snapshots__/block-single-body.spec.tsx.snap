// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BlockSingleBody /> renders correctly 1`] = `
Array [
  <section
    className="sc-bZQynM kUwNZg"
  >
    <ul
      className="sc-gzVnrw gjsEYR"
    >
      <div
        className="sc-ifAKCX kEvjdA sc-htoDjs bdMEGW"
      >
        <div
          className="sc-htpNat hwnZXv"
          size="sm"
        >
          Size
        </div>
        <div
          className="sc-EHOje foZbCf"
        >
          <div
            className="sc-bdVaJa htfwrO"
            size="lg"
          >
            <div
              className="sc-bwzfXH eAbjIM"
            >
               
            </div>
          </div>
        </div>
      </div>
      <div
        className="sc-ifAKCX kEvjdA sc-htoDjs bdMEGW"
      >
        <div
          className="sc-htpNat hwnZXv"
          size="sm"
        >
          Difficulty
        </div>
        <div
          className="sc-EHOje foZbCf"
        >
          <div
            className="sc-bdVaJa htfwrO"
            size="lg"
          >
            <div
              className="sc-bwzfXH dXrMlk"
            >
               
            </div>
          </div>
        </div>
      </div>
      <div
        className="sc-ifAKCX kEvjdA sc-htoDjs bdMEGW"
      >
        <div
          className="sc-htpNat hwnZXv"
          size="sm"
        >
          Total Difficulty
        </div>
        <div
          className="sc-EHOje foZbCf"
        >
          <div
            className="sc-bdVaJa htfwrO"
            size="lg"
          >
            <div
              className="sc-bwzfXH dTNUzo"
            >
               
            </div>
          </div>
        </div>
      </div>
      <div
        className="sc-ifAKCX kEvjdA sc-htoDjs bdMEGW"
      >
        <div
          className="sc-htpNat hwnZXv"
          size="sm"
        >
          Gas Used
        </div>
        <div
          className="sc-EHOje foZbCf"
        >
          <div
            className="sc-bdVaJa htfwrO"
            size="lg"
          >
            <div
              className="sc-bwzfXH dXrMlk"
            >
               
            </div>
          </div>
        </div>
      </div>
      <div
        className="sc-ifAKCX kEvjdA sc-htoDjs bdMEGW"
      >
        <div
          className="sc-htpNat hwnZXv"
          size="sm"
        >
          Gas Limit
        </div>
        <div
          className="sc-EHOje foZbCf"
        >
          <div
            className="sc-bdVaJa htfwrO"
            size="lg"
          >
            <div
              className="sc-bwzfXH eAbjIM"
            >
               
            </div>
          </div>
        </div>
      </div>
    </ul>
  </section>,
  <section
    className="sc-gZMcBi hWXaJA"
  >
    <ul
      className="sc-gqjmRU eoNvgc"
    >
      <div
        className="sc-dnqmqq lbIpyM sc-VigVT llXpEY"
      >
        <div
          className="sc-htpNat hwnZXv"
          size="sm"
        >
          Size
        </div>
        <div
          className="sc-iwsKbI ibFzST"
        >
          <div
            className="sc-bdVaJa htfwrO"
            size="lg"
          >
            <div
              className="sc-bwzfXH eAbjIM"
            >
               
            </div>
          </div>
        </div>
      </div>
      <div
        className="sc-dnqmqq lbIpyM sc-VigVT llXpEY"
      >
        <div
          className="sc-htpNat hwnZXv"
          size="sm"
        >
          Difficulty
        </div>
        <div
          className="sc-iwsKbI ibFzST"
        >
          <div
            className="sc-bdVaJa htfwrO"
            size="lg"
          >
            <div
              className="sc-bwzfXH dXrMlk"
            >
               
            </div>
          </div>
        </div>
      </div>
      <div
        className="sc-dnqmqq lbIpyM sc-VigVT llXpEY"
      >
        <div
          className="sc-htpNat hwnZXv"
          size="sm"
        >
          Total Difficulty
        </div>
        <div
          className="sc-iwsKbI ibFzST"
        >
          <div
            className="sc-bdVaJa htfwrO"
            size="lg"
          >
            <div
              className="sc-bwzfXH dTNUzo"
            >
               
            </div>
          </div>
        </div>
      </div>
      <div
        className="sc-dnqmqq lbIpyM sc-VigVT llXpEY"
      >
        <div
          className="sc-htpNat hwnZXv"
          size="sm"
        >
          Gas Used
        </div>
        <div
          className="sc-iwsKbI ibFzST"
        >
          <div
            className="sc-bdVaJa htfwrO"
            size="lg"
          >
            <div
              className="sc-bwzfXH dXrMlk"
            >
               
            </div>
          </div>
        </div>
      </div>
      <div
        className="sc-dnqmqq lbIpyM sc-VigVT llXpEY"
      >
        <div
          className="sc-htpNat hwnZXv"
          size="sm"
        >
          Gas Limit
        </div>
        <div
          className="sc-iwsKbI ibFzST"
        >
          <div
            className="sc-bdVaJa htfwrO"
            size="lg"
          >
            <div
              className="sc-bwzfXH eAbjIM"
            >
               
            </div>
          </div>
        </div>
      </div>
    </ul>
  </section>,
]
`;
